djdt_settings.py

```diff
@@ -0,0 +1,32 @@
+# To enable the Django Debug Toolbar for local dev add the following line to
+# your local_settings.py file:
+# from djdt_settings import *
+
+from settings import *  # noqa
```

Comment:
The `settings.py` file is in the root, which isn't a package, so we can't do that.